share/smack/top.py

```diff
@@ -297,6 +298,17 @@ def smack_headers():
 def smack_lib():
   return os.path.join(smack_root(), 'share', 'smack', 'lib')
 
+def objc_clang_compile_command(args, lib = False):
+  cmd = default_clang_compile_command(args,lib)
+  if sys.platform in ['linux', 'linux2']:
+    objc_flags = try_command(['gnustep-config', '--objc-flags'])
+    cmd += objc_flags.split()
+  elif sys.platform == 'darwin':
+    sys.exit("Objective-C not yet supported on macOS")
```

Comment:
We should merge the two else branches as they mean the same.

Reply:
If I remember right, this was suggested by Zvonimir or Micheal Emmi, but the comments disappeared after a force push... I can merge them if we want.